great_expectations/core/async_executor.py

```diff
@@ -81,7 +81,6 @@ def __exit__(self, exc_type, exc_value, traceback):
         self.shutdown()
         # Do not do use the context manager exception arguments, in order to get the desired default behavior where any
         # exceptions are raised here. More info at https://docs.python.org/3.9/reference/datamodel.html#object.__exit__.
-        AbstractContextManager.__exit__(self, exc_type, exc_value, traceback)
```

Comment:
Wups, I didn't mean to include this line. This is removed in subsequent commit 367f14e.